exercises/practice/matching-brackets/.meta/example.cairo

```diff
@@ -0,0 +1,51 @@
+pub fn is_paired(value: ByteArray) -> bool {
+    let mut stack: Array<u8> = Default::default();
+    let mut pairs = paired_brackets();
+
+    let mut i = 0;
+    let mut result = true;
+    while i < value
+        .len() {
+            let chr = value[i];
+            if chr == '{' || chr == '[' || chr == '(' {
+                stack.append(chr);
+            } else if chr == '}' || chr == ']' || chr == ')' {
+                if stack.is_empty() {
+                    result = false;
+                    break;
+                }
+
+                let mut popped: Array<u8> = Default::default();
+                let mut i = 0;
+                let mut stop = stack.len() - 1;
+                while i < stop {
+                    popped.append(*stack[i]);
```

Comment:
The example doesn't compile:
```console
error: Variable was previously moved.
 --> /tmp/tmp.dfvn7nEzZd/src/lib.cairo:26:29
                let last = *stack[stack.len() - 1];
                            ^***^
note: variable was previously used here:
  --> /tmp/tmp.dfvn7nEzZd/src/lib.cairo:22:36
                    popped.append(*stack[i]);
                                   ^***^
note: Trait has no implementation in context: core::traits::Copy::<core::array::Array::<core::integer::u8>>

error: could not compile `matching_brackets` due to previous error
Error: `scarb metadata` exited with error
```
The reason this wasn't caught by the CI/CD is due to the script that just "swallows" errors in individual tests, and just marks the job as "successful". This is my bad, will address this immediately.

Reply:
The test suite passes locally on 2.7.0 so I'll work on this some more.

Reply:
[CI tests are currently running with Scarb v2.6.3](https://github.com/exercism/cairo/blob/main/.github/workflows/test.yml#L41)